### PR TITLE
ISPN-5142 Cross site state transfer - retry mechanism not working correc...

### PR DESCRIPTION
**Pull request by pruivo**
...tly

https://issues.jboss.org/browse/ISPN-5142

### CODE REVIEW

File: core/src/main/java/org/infinispan/remoting/inboundhandler/AbstractDelegatingHandler.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
+package org.infinispan.remoting.inboundhandler;
+
+import org.infinispan.commands.remote.CacheRpcCommand;
+
+/**
+ * Designed to be overwrite.
```
Comment (danberindei) [inline core/src/main/java/org/infinispan/remoting/inboundhandler/AbstractDelegatingHandler.java]: I assume you mean overridden, but this is an abstract class so it doesn't say anything new...

File: core/src/test/java/org/infinispan/xsite/statetransfer/failures/RetryMechanismTest.java [hunk 2]
```diff
@@ -0,0 +1,431 @@
...
+ * @author Pedro Ruivo
+ * @since 7.1
+ */
+@Test(groups = "functional", testName = "xsite.statetransfer.failures.RetryMechanismTest")
+public class RetryMechanismTest extends AbstractTopologyChangeTest {
+
+   private static final String VALUE = "value";
+
+   /**
+    * Simple scenario where the primary owner throws an exception. The NYC master site should resend the data to the
```
Comment (danberindei) [inline core/src/test/java/org/infinispan/xsite/statetransfer/failures/RetryMechanismTest.java]: s/master site/site master/

**Comment by pruivo** [comment]
PR updated @danberindei

**Comment by danberindei** [comment]
Integrated, thanks Pedro!

**Pull request closed without merging**